src/boundary/boundarycondition.cpp

```diff
@@ -197,7 +197,7 @@ void BoundaryCondition::setCellBVOLDerivativesToZero(FsGrid<std::array<Real, fsg
  * closest boundarytype::NOT_BOUNDARY cell.
  * \param copyMomentsOnly If true, do not touch velocity space.
  */
-void BoundaryCondition::vlasovBoundaryCopyFromTheClosestNbr(
+void BoundaryCondition::vlasovBoundaryCopyFromClosestNbr(
```

Comment:
This was because we have two functions, one for "the closest" i.e. one cell, one for all "closest neighbors", and to make it more obvious "The" was included in the name.

Reply:
I noticed that, but since the other one is named `vlasovBoundaryCopyFromAllClosestNbrs` with a plural form, I would hope this is ok. There are other place where I see, e.g. `SysBoundaryConditions` vs `SysBoundaryCondition`, `SysBoundaries` vs `SysBoundary` etc.

Reply:
I think this one is OK. I picked up on it because I was seeing a single commit's worth of changes.

Places with e.g. sysBoundaries (a vector containing all of them) vs. the class name SysBoundary should be correct and consistent, but do point out if there's unclear things for sure.